build.gradle

```diff
@@ -58,7 +58,7 @@ allprojects {
     // with Navigation Native forced to be some snapshot version
     // if you need to use snapshots while development, just set `addSnapshotsRepo` to true manually
     def addSnapshotsRepo = project.hasProperty('ALLOW_SNAPSHOT_REPOSITORY') ? project.property('ALLOW_SNAPSHOT_REPOSITORY') : (System.getenv("ALLOW_SNAPSHOT_REPOSITORY")?.toBoolean() ?: false)
-    if (addSnapshotsRepo) {
+    if (true) {
```

Comment:
uses to allow snapshots only, will be reverted when the next mapbox-java will be released

Reply:
v6.5.0-beta.5 should be available already.